### PR TITLE
Update docs for pre-commit with plugins

### PR DESCRIPTION
**Pull request by sosukesuzuki**
## Description

Closes https://github.com/prettier/prettier/issues/15388

Since 3.0, we need to write plugin paths to `.prettierrc.js` for pre-commit with plugins. So I think we should mention about it in docs.

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: docs/precommit.md [hunk 1]
```diff
@@ -69,6 +69,8 @@ Copy the following config into your `.pre-commit-config.yaml` file:
 
 Read more at [mirror of prettier package for pre-commit](https://github.com/pre-commit/mirrors-prettier) and the [pre-commit](https://pre-commit.com) website.
 
+If you use Prettier 3.0 or later with plugins, you need to write the path to plugins in `.prettierrc.js`. See https://github.com/sosukesuzuki/prettier-precommit-with-plugins-example for details.
```
Comment (fisker) [inline docs/precommit.md:72]: The link is broken.

**Comment by fisker** [comment]
Have you investigate why it doesn't work? I don't think #15388 is the same case as https://github.com/prettier/prettier/issues/15085#issuecomment-1636993346

**Comment by fisker** [comment]
Even if it requires to use the path to plugin, I think this should document at https://github.com/pre-commit/mirrors-prettier instead.

**Comment by sosukesuzuki** [comment]
> Have you investigate why it doesn't work? I don't think https://github.com/prettier/prettier/issues/15388 is the same case as https://github.com/prettier/prettier/issues/15085#issuecomment-1636993346

Honestly, I haven't looked into it much. Do you have any idea what the cause is?

> Even if it requires to use the path to plugin, I think this should document at https://github.com/pre-commit/mirrors-prettier instead.

Ok, you are right. Anyway this PR will be closed.

**Comment by fisker** [comment]
> Honestly, I haven't looked into it much. Do you have any idea what the cause is?

I have not got time to look into it.

**Pull request closed without merging**